_includes/js/custom.js

```diff
@@ -149,67 +149,101 @@ jQuery(document).ready(function() {
+                tracker =`<a class="btn btn-default" href="${data[i]["tracker"]}" aria-label="Go to the tracker for ${title}" title="Go to the tracker for ${title}">
                                 <span class="glyphicon glyphicon-comment" aria-hidden="true"></span>
                             </a>`;
+            } else {
+                tracker = `
+                        <a class="btn btn-default disabled">
+                            <span class="glyphicon glyphicon-comment" aria-hidden="true"></span>
+                        </a>`;
             }
             if (data[i].hasOwnProperty("domain") && data[i]['domain'] !== undefined) {
                 domainInner[0] = data[i]['domain'];
             }
             if (description !== undefined && description.toString().length > 140) {
-                description = description.toString().slice(0, 140) + '...'
+                description = description.toString().slice(0, 140).trim() + '...'
             }
             if (data[i]["contact"]) {
                 contact =`
-                        <a class="btn btn-default btn-sm" href="mailto:${data[i]["contact"]["email"]}" aria-label="Send an email to ${title}"
+                        <a role="button" class="btn btn-default" href="mailto:${data[i]["contact"]["email"]}" aria-label="Send an email to ${title}"
                            title="Send an email to ${title}">\
                             <span class="glyphicon glyphicon-envelope" aria-hidden="true"></span>
                         </a>`;
+            } else {
+                contact = `
+                        <a role="button" class="btn btn-default disabled">
+                            <span class="glyphicon glyphicon-envelope" aria-hidden="true"></span>
+                        </a>`;
             }
             if (data[i]["publications"] && data[i]["publications"].length > 0) {
                 publication = `
-                            <a class="btn btn-default btn-sm" href="${data[i]["publications"][0]["id"]}" aria-label="View the primary publication for ${title}" title="View the primary publication for ${title}">
+                            <a role="button" class="btn btn-default" href="${data[i]["publications"][0]["id"]}" aria-label="View the primary publication for ${title}" title="View the primary publication for ${title}">
                                 <span class="glyphicon glyphicon-book" aria-hidden="true"></span>
                             </a>`;
+            } else {
+                publication = `
+                        <a role="button" class="btn btn-default disabled">
+                            <span class="glyphicon glyphicon-book" aria-hidden="true"></span>
+                        </a>`;
             }
-            if (activity_status === "inactive") {
+            if (activity_status === "inactive" || activity_status === "orphaned") {
                 is_inactive = "inactive_row";
+                is_obsolete = `(${activity_status})`
```

Comment:
What exactly does this change do?

Reply:
I didn't change the variable name, but this makes the labels for orphaned and inactive show up.

| Before | After |
|:------:|:----:|
| <img width="1466" alt="Screen Shot 2022-08-25 at 09 57 06" src="https://user-images.githubusercontent.com/5069736/186608735-2994fd22-8ac4-43ea-8e4c-bb8a95b99c0e.png">  | <img width="1466" alt="Screen Shot 2022-08-25 at 09 57 19" src="https://user-images.githubusercontent.com/5069736/186608794-f99faf1b-1a92-4803-8b71-8f666b5a0921.png"> |